Warning! here be dragons.

Because planex processes the patchqueue when creating the srpm, a regular
.gitarchive-info in the patchqueue tarball isn't available to the build
environment.  Fake it up in a patch file to appear as if it were created in
the base tarball.

diff --git a/.gitarchive-info-pq b/.gitarchive-info-pq
new file mode 100644
index 0000000..83e5b86
--- /dev/null
+++ b/.gitarchive-info-pq
@@ -0,0 +1,2 @@
+Changeset: 9f332f65f7606c98e4fa190fe6cddabc4b237adc
+Commit date: Mon, 18 Mar 2019 15:06:21 +0000
